document/FileFormat.mdk

```diff
@@ -2696,31 +2703,318 @@ The tool might provide a "related location" (&sect;[#result-relatedLocations]) w
+|                  | wishes to represent.        |                             |                             |
+|------------------|-----------------------------|-----------------------------|-----------------------------|
+| `assignment`     | Recommended: The            | Recommended: A              | Optional: the names and     |
+|                  | name of the variable        | string representation       | values of the variables     |
+|                  | being assigned to.          | of the value                | participating in the        |
+|                  |                             | assigned to the             | expression on the right-    |
+|                  |                             | target variable.            | hand side of the            |
+|                  |                             |                             | assignment.                 |
+|------------------|-----------------------------|-----------------------------|-----------------------------|
+| `branch`         | Recommended: If the         | Optional: If the branch     | Optional: If the branch is  |
+|                  | target of the branch is a   | is the result of a test,    | the result of a test,       |
+|                  | named label, in which       | in which case, a string     | in which case, the names    |
+|                  | case, the name of           | representation of the       | and values of the variables |
+|                  | the label.                  | Boolean                     | participating in the        |
+|                  | Optional: If the target of  | value of the test           | expression being tested.    |
+|                  | the branch is a known       | condition (`"true"`         | Absent: If the branch is    |
+|                  | address or offset, in       | or `"false"`).              | unconditional.              |
+|                  | which case, a               | Absent: If the branch is    |                             |
+|                  | hexadecimal                 | unconditional.              |                             |
+|                  | string representation of    |                             |                             |
+|                  | that address or offset.     |                             |                             |
+|                  | Absent: If the target of    |                             |                             | 
+|                  | the branch is not           |                             |                             |
+|                  | known.|                     |                             |                             |
+|------------------|-----------------------------|-----------------------------|-----------------------------|
+| `call`           | Recommended: The            | Optional: An array of       | Optional: The names and     |
```

Comment:
I added some text for this, and another example.

---
In reply to: [77425380](https://github.com/sarif-standard/sarif-spec/pull/264#discussion_r77425380) [](ancestors = 77425380)